src/router/index.js

```diff
@@ -0,0 +1,128 @@
+import {getRoutableSignals, flattenConfig} from './utils'
+import urlMapper from 'url-mapper'
+
+let addressbar
+try {
+  addressbar = require('addressbar')
+} catch (e) {
+  addressbar = {
+    pathname: '/',
+    value: '',
+    origin: '',
+    on() {},
+    removeListener() {}
+  }
+}
+
+export default function Router(options = {}) {
```

Comment:
Might be better to use base router without default mapper. It will free router from knowing mapper special options (like query), and also will allow to bring breaking changes into mapper without need to bump cerebral major version. It will also shrink a build size for ones who uses own mapper implementation.